index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" type="text/css" >
+    <script src="https://kit.fontawesome.com/8d0a2aa6d4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/home.css">
+    <title>Capstone project</title>
+</head>
+<body>
+    <header class="header">
+        <ul id="header-list">
+            <li class="header-item icon"><a href="#"><i class="fa fa-facebook"></i></a></li>
+            <li class="header-item icon"><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+            <li class="header-item"><a href="#">English</a></li>
+            <li class="header-item"><a href="#">My Page</a></li>
+            <li class="header-item"><a href="#">Logout</a></li>
+        </ul>
+    </header>
+    <div class="head-container">
+        <div class="head-image">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        <nav class="nav-container">
+            <div class="nav-logo">
+            <img src="images/LogoMakr-3POqWB.png" class="nav-logo-img" alt="home logo">
+            <a href="index.html" class="nav-head">Global Youth <br> Empowerment</a>
+            </div>
```

Comment:
Optional -
I strongly suggest you make a quick update to your logo to improve the UX of the webpage

Reply:
Hie  @alexander16108  , thank you for your feedback, could you please elaborate more on that point.

Reply:
Hi @Leeoasis, its an optional suggestion on changing the logo of your webpage because it looks rough and unappealing, so updating it or make a new on based on canvas templates